packages/adp-tooling/src/types.ts

```diff
@@ -52,6 +58,23 @@ export interface AdpWriterConfig {
     };
 }
 
+export interface FlpConfiguration {
+    title: string;
+    bspName: string;
+    languages: Language[];
+    inboundId: string;
```

Comment:
The type indicates that `inboundId` is mandatory but looking at the code it is not.